src/modules/common/components/Page.js

```diff
@@ -1,24 +1,48 @@
 // @flow
 
-import React from 'react'
-import Caption from './Caption'
+import * as React from 'react'
+import styled from 'styled-components'
 import RemoteContent from './RemoteContent'
+import Caption from './Caption'
+import type Moment from 'moment'
+import LastUpdateInfo from './LastUpdateInfo'
+
+const Thumbnail = styled.img`
+  display: flex;
+  width: 300px;
+  height: 300px;
+  margin: 10px auto;
+  padding-bottom: 10px;
+  object-fit: contain;
+`
 
 type PropsType = {|
   title: string,
+  thumbnail?: string,
   content: string,
-  onInternLinkClick: string => void,
-  hijackRegExp?: RegExp
+  lastUpdate: Moment,
```

Comment:
Please make lastUpdate optional - we might not want to show a lastUpdate on every Page (in the wohnraumbörse, we use the Page component and we probably don't want to show a last-update there)

Reply:
Done